beaver/transports/rabbitmq_transport.py

```diff
@@ -93,7 +94,7 @@ def _on_connection_open_error(self,non_used_connection=None,error=None):
 
     def _on_connection_closed(self, connection, reply_code, reply_text):
         self._channel = None
-        if self._connection._closing:
+        if self._connection.is_closing:
```

Comment:
Probably worth supporting both old and new versions of pika. Pretty easy to check which version in the init of the transport and then just use that one here.